openstudiocore/src/airflow/contam/ForwardTranslator.cpp

```diff
@@ -434,10 +434,18 @@ boost::optional<contam::IndexModel> ForwardTranslator::translateModel(model::Mod
   m_logSink.setThreadId(QThread::currentThread());
   m_logSink.resetStringStream();
 
-  m_prjModel.read(std::string(":/templates/template.prj"));
-  if (!m_prjModel.valid()) {
-    return boost::none;
+  {
+    QFile f(":/templates/template.prj");
```

Comment:
I had to explicitly change this to create a QFile because it (`read` function) was previously creating a QFile internally and I removed that logic from PrjModel. So this limits the use of QFile to exactly the one function that cares about it.